@navikt/core/react/src/portal/Portal.tests.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import type { StoryObj } from "@storybook/react/*";
+import React from "react";
+import { expect, userEvent, within } from "storybook/test";
+import { Provider } from "../provider";
+import { Portal } from "./Portal";
+
+export default {
+  title: "Utilities/Portal/Tests",
+  parameters: {
+    chromatic: { disable: true },
+  },
+};
+
+type Story = StoryObj<typeof PortalComponent>;
+
+export const CustomContainer: Story = {
+  render: () => {
+    const [customRoot, setCustomRoot] = React.useState<HTMLElement | null>(
+      null,
+    );
+
+    return (
+      <div>
+        <div data-testid="custom-root" ref={setCustomRoot} />
+        <PortalComponent rootElement={customRoot} />
+      </div>
+    );
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const customRoot = canvas.getByTestId("custom-root");
+
+    const portalNode = canvas.getByTestId("portal-node");
+
+    expect(portalNode.parentElement).toBe(customRoot);
+  },
+};
+
+export const CustomContainerFromProvider: Story = {
+  render: () => {
+    const [customRoot, setCustomRoot] = React.useState<HTMLElement | null>(
+      null,
+    );
+
+    return (
+      <Provider rootElement={customRoot ?? undefined}>
+        <div data-testid="custom-root" ref={setCustomRoot} />
+        <PortalComponent />
+      </Provider>
+    );
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const customRoot = canvas.getByTestId("custom-root");
+
+    const portalNode = canvas.getByTestId("portal-node");
+
+    expect(portalNode.parentElement).toBe(customRoot);
+  },
+};
+
+export const AllowContainerChange: Story = {
+  render: () => {
+    const [customRootOne, setCustomRootOne] =
+      React.useState<HTMLElement | null>(null);
+    const [customRootTwo, setCustomRootTwo] =
+      React.useState<HTMLElement | null>(null);
+    const [activated, setActivated] = React.useState<boolean>(false);
+
+    return (
+      <div>
+        <button onClick={() => setActivated(true)}>Change root</button>
```

Comment:
Allow manual testing :)

```suggestion
        <button onClick={() => setActivated(!activated)}>Change root</button>
```